ui/test-resources/mapping/atlasmapping-expr-prop.json

```diff
@@ -204,6 +202,100 @@
           "userCreated" : false
         } ],
         "id" : "mapping.983999"
+      }, {
+        "jsonType" : "io.atlasmap.v2.Mapping",
+        "expression" : "repeat( count( ${JSONSchemaSource:/addressList<>/city}), ${DOC.Constants:/const-str})",
+        "inputFieldGroup" : {
+          "jsonType" : "io.atlasmap.v2.FieldGroup",
+          "actions" : [ ],
+          "field" : [ {
+            "jsonType" : "io.atlasmap.json.v2.JsonField",
+            "docId" : "JSONSchemaSource",
+            "path" : "/addressList<>/city",
+            "fieldType" : "STRING",
+            "name" : "city"
+          }, {
+            "jsonType" : "io.atlasmap.v2.ConstantField",
+            "value" : "const-str",
+            "docId" : "DOC.Constants",
+            "path" : "/const-str",
+            "fieldType" : "STRING"
+          } ]
+        },
+          "inputField" : [ ],
+          "outputField" : [ {
+            "jsonType" : "io.atlasmap.xml.v2.XmlField",
+            "actions" : [ ],
+            "docId" : "XMLInstanceSource",
+            "path" : "/ns:XmlOE/ns:Address/ns:addressLine1",
+            "fieldType" : "STRING",
+            "name" : "addressLine1",
+            "userCreated" : false
+          } ],
+          "id" : "mapping.983900"
+      }, {
+        "jsonType" : "io.atlasmap.v2.Mapping",
+        "expression" : "select( filter( ${JSONSchemaSource:/addressList<>}), ${/street} != 'Boston'), ${/state})",
+        "inputFieldGroup" : {
+          "jsonType" : "io.atlasmap.v2.FieldGroup",
+          "actions" : [ ],
+          "field" : [ {
+            "jsonType" : "io.atlasmap.json.v2.JsonField",
+            "docId" : "JSONSchemaSource",
+            "path" : "/addressList<>/street",
```

Comment:
It seems to be wrong - as the expression refers to `${JSONSchemaSource:/addressList<>}`, the `inputFieldGroup` has to contain `${JSONSchemaSource:/addressList<>}` with subsequent `${/street}` filled as a child.

Reply:
ok - will need to verify with the serializer.